src/components/SpaceSidebarFooter.vue

```diff
@@ -0,0 +1,62 @@
+<script setup>
+defineProps({
+  space: Object
+});
+</script>
+
+<script>
+export default {
+  data () {
+    return {
+      socials: [
+        {
+          url: 'https://github.com/',
+          spaceAttribute: 'github',
+          icon: 'github',
+        },
+        {
+          url: 'https://twitter.com/',
+          spaceAttribute: 'twitter',
+          icon: 'twitter',
+        },
+        {
+          url: 'https://discord.gg/',
+          spaceAttribute: 'discord',
+          icon: 'discord',
+        },
+      ]
+    }
+  },
+  computed: {
+    spaceSocials() {
+      return this.socials.filter(social => this.space[social.spaceAttribute])
+    }
+  }
+}
+</script>
+
+<template>
+  <div
+    v-if="space && spaceSocials.length"
+    class="text-center border-t bg-skin-header-bg mt-3"
+  >
+    <div>
+      <h3 class="my-2 mx-2 flex justify-center items-center space-x-3">
```

Comment:
What is the reason to use an `h3` here? The icon don't seem to be vertically centered properly

Reply:
shouldnt have been there